### PR TITLE
add an actor to report changes in wireshark

### PR DESCRIPTION
**Pull request by mruprich**
Just some small changes have been made in the options used with wireshark. Not much to do here, just report the changes.

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+class CheckWireshark(Actor):
+    """
+    Report a couple of changes in tshark usage
+    """
+
+    name = 'check_wireshark'
+    consumes = (InstalledRedHatSignedRPM, )
+    produces = (Report, )
+    tags = (FactsPhaseTag, IPUWorkflowTag)
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py]:
```suggestion
    tags = (ChecksPhaseTag, IPUWorkflowTag)
```

File: repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+from leapp.actors import Actor
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py]:
```suggestion
from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
```

File: repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.models import Report
+from leapp.reporting import create_report
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py]: Fix order of imports.

File: repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    name = 'check_wireshark'
+    consumes = (InstalledRedHatSignedRPM, )
+    produces = (Report, )
+    tags = (FactsPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+		if has_package(InstalledRedHatSignedRPM, 'wireshark'):
+			create_report([
+				reporting.Title('Changed cli option'),
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py]: Based on the title, it is not clear about what the report is. From the info it seems that this is about *tshark* utility. So something like: *tshark: changed cli options*. same below.

File: repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+					'of your scripts, please remove it.'),
+				reporting.Severity(reporting.Severity.LOW),
+				reporting.Tags([reporting.Tags.UPGRADE_PROCESS, reporting.Tags.MONITORING]),
+			])
+
+			create_report([
+				reporting.Title('Changed cli output'),
+				reporting.Summary(
+					'When using -H option with capinfos, the output no longer shows '
+					'SHA1, RIPEMD160 and MD5 hashes. Now it shows SHA256, RIPEMD160 and SHA1 hashes. '
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py]: is it about orther? you write that it shows and doesn't show SHA1 hashes in the same time.

**Comment by pirat89** [comment]
@mruprich Btw, wait with changes. We have currently discussions (with force majeure) whether to provide such information to user or not. just inform you about that so you do not spent on that additional time for nothing in the end. I will let you know.

**Comment by mruprich** [comment]
Ok, I will wait, thanks for the feedback.

**Comment by mruprich** [comment]
Did you reach any decision?

**Comment by pirat89** [comment]
@mruprich Hi, sorry but not yet. We had couple of PTOs and sicks and now there is @devconf time. I will raise that during next week in the worst.

File: repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+			create_report([
+				reporting.Title('Changed cli option'),
+				reporting.Summary(
+					'The -C suboption for -N option for asynchronous DNS name resolution '
+					'has been completely removed from tshark. The reason for this is that '
+					'the asynchronous DNS resolution is now the only resolution available '
+					'so there is no need for -C. If you are using -NC with tshark in any '
+					'of your scripts, please remove it.'),
+				reporting.Severity(reporting.Severity.LOW),
+				reporting.Tags([reporting.Tags.UPGRADE_PROCESS, reporting.Tags.MONITORING]),
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py]: note: use SANITY tag.

**Comment by mruprich** [comment]
@pirat89 I included the changes from your notes. I also changed my tabbing to spaces, it looks better imo.

**Pull request merged**